client/components/jetpack/card/jetpack-free-card/button.tsx

```diff
@@ -82,7 +82,10 @@ const JetpackFreeCardButton: FC< JetpackFreeCardButtonProps > = ( {
 	const { site: url, ...restQueryArgs } = urlQueryArgs;
 	const startHref =
 		isJetpackCloud() && ! isSiteinContext
-			? addQueryArgs( { url, ...restQueryArgs }, `https://wordpress.com${ JPC_PATH_BASE }` )
+			? addQueryArgs(
+					{ url, ...restQueryArgs, plan: PLAN_JETPACK_FREE },
+					`https://wordpress.com${ JPC_PATH_BASE }`
+			  )
```

Comment:
Let's pass Jetpack Free slug to the Jetpack Connect flow.

Reply:
We can't rely on Local Storage since the user is jumping from one domain to another.